h2o-r/h2o-package/R/communication.R

```diff
@@ -749,6 +749,40 @@ h2o.clusterInfo <- function() {
   }
 }
 
+.h2o.translateJobType <- function(type) {
```

Comment:
Can you please add the same function to python as well, so that we have a parity?